pages/dkp/kaptain/2.1.0/tutorials/ngc/index.md

```diff
@@ -0,0 +1,120 @@
+---
+layout: layout.pug
+navigationTitle: NGC catalog
+title: NVIDIA NGC catalog
+menuWeight: 30
+excerpt: Use NVIDIA's NGC catalog with Kaptain
+beta: false
+enterprise: false
+---
+
+Kaptain supports the usage of Nvidia's NGC catalog in networked environments, which you can access and set up via the Kubeflow UI. The NGC catalog includes containers, charts, pre-trained models, toolkits and more, which are optimized for GPU deployment and can now be used to run AI workflows in Kaptain. You can find all the available resources for your AI/ML workflow in the [NVIDIA NGC catalog][NGC_catalog].
+
+## Prerequisites
+
+-   You have [deployed Kaptain into a cluster](../../install/).
+
+-   You have a Kubeflow profile, and [permissions to create notebooks](../../user-management/).
+
+-   You have [installed JupyterLab](https://jupyterlab.readthedocs.io/en/2.2.x/getting_started/installation.html).
+
+-   You have permissions to access the [NGC component](https://docs.nvidia.com/ngc/ngc-catalog-user-guide/index.html#registering-activating-ngc-account).
+
+## Prepare your Kubeflow user profile
+
+You do not need to download, nor install the NGC catalog. You will be able to access it by logging into your Kaptain instance. However, you have to set the default entry point for Jupyter Lab first.
+
+In your Kubeflow profile, create a custom configuration for a notebook server to launch in Jupyter Lab. Specify the `user-namespace` where you want to create the notebook.
+
+1.  Open the `PodDefault` YAML file:
+
+    <!-- TODO: command to "open" or "edit" the yaml -->
```

Comment:
@alembiewski is there a command to open the yaml for editing? Could you provide it here, please? 
or should we go for something similar to `kubectl apply` where you create and apply a resource in one step? 
It seems to me as if the `PodDefault` resource is created for this purpose specifically?

Reply:
They can use an text editor, there's no special command and I don't think we need to provide more detail here.

However, I think to keep it consistent with other places in our docs, this formulation might be more better:

Create or edit a 'PodDefault' YAML file: